content/docs/data/Import.md

```diff
@@ -0,0 +1,319 @@
+---
+title: 'Importing Data'
+meta_title: 'Appbase - Dashboard'
+meta_description: 'Import through GUI and command line sources such as Dashboard, ABC CLI, Rest APIs, and Zapier.'
+keywords:
+    - dataschema
+    - appbase
+    - import
+    - elasticsearch
+sidebar: 'docs'
+---
+
+You can bring your data from various sources into `appbase.io` database using one or many of these methods.
+
+<div class="grid-integrations-index mt4 mt6-l f8">
+    <a class="bg-white shadow-2 box-shadow-hover shadow-2-hover  br4 db flex flex-column justify-between items-center middarkgrey pa2 pt5 pb5 tdn tc" style="box-shadow: 0 0 5px rgba(0,0,0,.02), 0 5px 22px -8px rgba(0,0,0,.1);    word-break: normal;cursor: pointer; padding: 2rem; height: 120px;width:120px;" href="#importing-through-dashboard">
+		<img class="w10 mb1" src="https://i.imgur.com/qXDGYSX.png" />
+		Dashboard
+	</a>
+	<a class="bg-white shadow-2 box-shadow-hover shadow-2-hover  br4 db flex flex-column justify-between items-center middarkgrey pa2 pt5 pb5 tdn tc" style="box-shadow: 0 0 5px rgba(0,0,0,.02), 0 5px 22px -8px rgba(0,0,0,.1);    word-break: normal;cursor: pointer; padding: 2rem; height: 120px;width:120px;" href="#importing-through-abc-cli">
+    		<img class="w10 mb1" src="https://user-images.githubusercontent.com/4047597/29240054-14e0e19a-7f7b-11e7-898b-ba6bad756b1d.png" />
+    		ABC CLI
+    </a>
+    <a class="bg-white shadow-2 box-shadow-hover shadow-2-hover  br4 db flex flex-column justify-between items-center middarkgrey pa2 pt5 pb5 tdn tc" style="box-shadow: 0 0 5px rgba(0,0,0,.02), 0 5px 22px -8px rgba(0,0,0,.1);    word-break: normal;cursor: pointer; padding: 2rem; height: 120px;width:120px;" href="#rest-api">
+        		<img class="w10 mb1" src="https://i.imgur.com/nKKQLXb.png" />
+        		Rest API
+     </a>
+	<a class="bg-white shadow-2 box-shadow-hover shadow-2-hover  br4 db flex flex-column justify-between items-center middarkgrey pa2 pt5 pb5 tdn tc" style="box-shadow: 0 0 5px rgba(0,0,0,.02), 0 5px 22px -8px rgba(0,0,0,.1);    word-break: normal;cursor: pointer; padding: 2rem; height: 120px;width:120px;" href="#zapier">
+		<img class="w10 mb1" src="https://i.imgur.com/Z9ahZbM.png" />
+		Zapier
+	</a>
+</div>
+
+---
+
+##Importing through Dashboard
```

Comment:
@ShahAnuj2610 We can show the dashboard importer screen UI at the start of the section here.